### ISSUE TITLE
Add a how-to for setting up conda on windows.

### ISSUE THREAD

**Issue by JoeZiminski** [neuroinformatics-unit/neuroinformatics-unit.github.io#303]
There are some weird things about setting up conda on windows that have caused problems for new users and are basically impossible to debug without a lot of googling. I think the beow should work for `cmd` but definately work for powershell.

1) Set the conda directory in environment variables so powershell can recognise it. 
https://stackoverflow.com/questions/44597662/conda-command-is-not-recognized-on-windows-10

2) Set the powershell permissions to allow conda to function

https://stackoverflow.com/questions/62082873/conda-not-activated-in-power-shell

3) `conda init powershell`

**Comment by niksirbi** [comment]
For a more up-to-date guide on how to resolve this, see dropdown in https://github.com/neuroinformatics-unit/course-animals-in-motion/issues/50

**Comment by adamltyson** [comment]
With the rise of uv etc, is this still necessary? It seems more useful to have in specific course materials than here?

**Comment by niksirbi** [comment]
I mostly encounter this during teaching. We already have an issue about this on https://github.com/neuroinformatics-unit/course-animals-in-motion/issues/50, so fine for me to handle there. But not sure if it's also relevant for other courses/handbooks?

I think the extracellular ephys course already prioritises `uv`, so it's mostly BrainGlobe that's in need of conda (and its tricky Windows setup).

**Comment by adamltyson** [comment]
Yeah I think we should just keep this in the relevant course materials. If BrainGlobe needs to document it, it should be in the BrainGlobe course docs or website.
